### PR TITLE
[artifactory-ha] Fix PodDisruptionBudget selector issue

### PR DESCRIPTION
**Pull request by rahulsadanandan**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] CHANGELOG.md updated
- [x] Variables and other changes are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[artifactory]`)

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Review by eldada — APPROVED** [review]

**Review by peleduri — APPROVED** [review]

File: stable/artifactory-ha/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # JFrog Artifactory-ha Chart Changelog
 All changes to this chart will be documented in this file.
 
+## [4.3.3] - Oct 15, 2020
+* Fix PodDisruptionBudget selector issue.
```
Comment (chukka) [inline stable/artifactory-ha/CHANGELOG.md:5]: @rahulsadanandan  can you add small description of what the issue is ?

Comment (rahulsadanandan) [reply stable/artifactory-ha/CHANGELOG.md]: Its done now

**Review by chukka — APPROVED** [review]

**Review by rimusz — APPROVED** [review]
LGTM

File: stable/artifactory-ha/templates/artifactory-node-pdb.yaml [hunk 2]
```diff
@@ -13,8 +14,7 @@ spec:
     matchLabels:
       component: {{ .Values.artifactory.name }}
       app: {{ template "artifactory-ha.name" . }}
-      {{- if eq .Values.artifactory.service.pool "members" }}
       role: {{ template "artifactory-ha.node.name" . }}
-      {{- end }}
       release: {{ .Release.Name }}
   minAvailable: {{ .Values.artifactory.node.minAvailable }}
+{{- end }}
```
Comment (eldada) [inline stable/artifactory-ha/templates/artifactory-node-pdb.yaml:20]: Add new line

**Comment by kristinnardal2** [comment]
I thought that this was a feature of the Helm chart.

Setting `artifactory.service.pool` to all would not only make sure that traffic is routed to both primary and member nodes but also that the pod disruption policy would make sure that there is always at least one node available, be it primary or member node.

There is a real use case in having a pod disruption budget being defined over both. The option to do that with the Helm chart has been removed with this change.

**Pull request merged**